src/api/product-filters.ts

```diff
@@ -62,11 +62,10 @@ export function parseProductFiltersFromUrlSearchParams(
     };
 }
 
-export function stringifyProductFiltersToUrlSearchParams({
-    minPrice,
-    maxPrice,
-}: IProductFilters): URLSearchParams {
-    const searchParams = new URLSearchParams();
+export function setProductFiltersToUrlSearchParams(
```

Comment:
It's ambiguous whether it's `setProductFilters` or `setUrlSearchParams`.

One alternative:
```
convertProductFiltersToUrlParams(filters): URLSearchParams
convertUrlParamsToProductFilters(params): IProductFilters

mergeUrlParams(a: URLSearchParams, b: URLSearchParams): URLSearchParams
```